spec/features/user_gets_rss_spec.rb

```diff
@@ -0,0 +1,28 @@
+feature "User gets RSS Feed" do
+
+  let!(:user){ Fabricate(:user) }
```

Comment:
`page.has_content?` isn't an assertion.  It won't cause the test to fail if it isn't present.

This file should have many `page.should have_content` assertions.